addon/components/layout/left-menu/component.js

```diff
@@ -6,6 +6,26 @@ export default Ember.Component.extend({
   classNames: ['__left-menu'],
 
   session: Ember.inject.service(),
+  currentUser: Ember.inject.service(),
+
+  hasFacade: false,
+  hasInbox: false,
+
+  init() {
+    this._super();
+    this.get('currentUser').fetch().then((payload) => {
+      if (payload.user.granted_scopes.includes('inbox_client')) {
+        this.set('hasInbox', true);
+      }
+
+      if (payload.user.granted_scopes.includes('facade_web')) {
+        this.set('hasFacade', true);
+      }
+    }).catch(() => {
+      this.set('hasFacade', true);
+      this.set('hasInbox', true);
```

Comment:
if we have a problem during the HTTP call or payload parsing i do prefer display all the links  over none of them.

Most of our customers will have access to both systems.